src/OpenLoco/GameCommands/LoadSaveQuit.cpp

```diff
@@ -0,0 +1,323 @@
+#include "../Audio/Audio.h"
+#include "../CompanyManager.h"
+#include "../Config.h"
+#include "../GameCommands.h"
+#include "../GameException.hpp"
+#include "../Graphics/Gfx.h"
+#include "../Input.h"
+#include "../Interop/Interop.hpp"
+#include "../MultiPlayer.h"
+#include "../Objects/ObjectManager.h"
+#include "../OpenLoco.h"
+#include "../S5/S5.h"
+#include "../Title.h"
+#include "../Tutorial.h"
+#include "../Ui/WindowManager.h"
+#include "../Ui/WindowType.h"
+
+#pragma warning(disable : 4702)
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    static loco_global<uint8_t, 0x00508F08> _game_command_nest_level;
+    static loco_global<uint16_t, 0x0050A002> _savePromptType;
+
+    static loco_global<char[256], 0x0050B745> _currentScenarioFilename;
+    static loco_global<char[512], 0x0112CE04> _savePath;
+
+    // 0x004416FF
+    static bool loadSaveGameOpen()
+    {
+        registers regs;
+        call(0x004416FF, regs);
+        return regs.eax;
+    }
+
+    // 0x00441843
+    static bool saveSaveGameOpen()
+    {
+        registers regs;
+        call(0x00441843, regs);
+        return regs.eax;
+    }
+
+    // 0x004417A7
+    static bool loadLandscapeOpen()
+    {
+        registers regs;
+        call(0x004417A7, regs);
+        return regs.eax;
+    }
+
+    // 0x00441993
+    static bool saveLandscapeOpen()
+    {
+        registers regs;
+        call(0x00441993, regs);
+        return regs.eax;
+    }
+
+    // 0x00441FA7
+    static bool sub_441FA7()
+    {
+        registers regs;
+        call(0x00441FA7);
+        return regs.eax != 0;
+    }
+
+    // 0x004424CE
+    static bool sub_4424CE()
+    {
+        registers regs;
+        call(0x004424CE);
+        return regs.eax != 0;
+    }
+
+    // 0x0043BFF8
+    static uint32_t loadGame()
+    {
+        GameCommands::do_21(1, 0);
+        Input::toolCancel();
+
+        if (isEditorMode() && loadLandscapeOpen())
```

Comment:
I'm not sure having the if like this is quite the same. If loadLandscapeOpen returns false this version will move to loadSaveGameOpen. This is different to vanilla where it would leave the if else chain.